tests/test_registration_embargoes.py

```diff
@@ -76,7 +78,8 @@ def test_embargo_initiator_has_backref(self):
         )
         self.registration.save()
         self.registration.reload()
-        assert_equal(len(self.user.embargo__embargoed), 1)
+        assert_equal(len(self.user.embargo__embargoed),
+            Embargo.find(Q('initiated_by', 'eq', self.user)).count())
```

Comment:
Yes; a necessary fix was made to the Sanctions factories which broke this test. This makes the test correct.